### PR TITLE
[SENP-826] feat(workflows): add docker tag workflow

### PR DESCRIPTION
**Pull request by jdassonvil**
This new workflow will be used to tag previously built docker image with the release version.

### CODE REVIEW

File: .github/workflows/docker-tag-v1.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        default: ""
+      docker_image_name:
+        type: string
+        required: true
+      docker_tag_from:
+        type: string
+        required: true
+      docker_tag_to:
+        type: string
+        required: true
```
Comment (rdelgatte) [inline .github/workflows/docker-tag-v1.yml:21]: Do we need to prefix these input variables name with `docker`? This looks redundant to me as we're in the context of a `Docker tag` action here :-)

File: .github/workflows/docker-tag-v1.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+on:
+  workflow_call:
+    inputs:
+      aws_region:
+        type: string
+        default: "eu-central-1"
+      ci_iam_role:
+        type: string
+        default: ""
```
Comment (rdelgatte) [inline .github/workflows/docker-tag-v1.yml:12]: I would move the optional inputs parameters after the required one, this is usually a good practice to ease readability / maintainability (IMHO)

**Review by rdelgatte — APPROVED** [review]
Minor suggestions for naming/readability

File: .github/workflows/docker-tag-v1.yml (hunk 1 above)
Comment (jdassonvil) [reply .github/workflows/docker-tag-v1.yml:21]: No we don't, we just need to be consistent

File: .github/workflows/docker-tag-v1.yml (hunk 2 above)
Comment (jdassonvil) [reply .github/workflows/docker-tag-v1.yml:12]: ok I don't have a strong opinion on that
most of our workflows parameters are sorted in alphabetical order
I think that's a good suggestion but we need to that consistently in all workflows

Comment (jdassonvil) [reply .github/workflows/docker-tag-v1.yml:12]: I have created a dedicated card for this:
https://www.notion.so/sencrop/Reorder-parameters-on-Github-workflows-63fd0e85e1294399958faeb287fedc27?pvs=4

File: .github/workflows/docker-tag-v1.yml (hunk 1 above)
Comment (jdassonvil) [reply .github/workflows/docker-tag-v1.yml]: https://github.com/sencrop/github-workflows/pull/89

**Pull request merged**